Got error when trying to build open-ce with redhat 7.6alt on an AC922

**Describe the bug**
The build failed.
make[2]: *** [Source/CursesDialog/CMakeFiles/ccmake.dir/build.make:282: bin/ccmake] Error 1
make[1]: *** [CMakeFiles/Makefile2:1985: Source/CursesDialog/CMakeFiles/ccmake.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
Traceback (most recent call last):
  File "/home/builder/open-ce/open-ce/build_feedstock.py", line 134, in build_feedstock_from_command
    conda_build.api.build(os.path.join(os.getcwd(), recipe['path']),
  File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3008, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2131, in build
    utils.check_call_env(cmd, env=env, rewrite_stdout_env=rewrite_env,
  File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 410, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 390, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/cmake_1610144864674/work/conda_build.sh']' returned non-zero exit status 2.
[OPEN-CE-ERROR-11] Unable to build recipe: cmake-feedstock
[OPEN-CE-ERROR-11] Unable to build recipe: cmake-feedstock
Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/cmake_1610144864674/work/conda_build.sh']' returned non-zero exit status 2.
cd $SRC_DIR/Source && $BUILD_PREFIX/bin/powerpc64le-conda_cos7-linux-gnu-c++  -DLIBARCHIVE_STATIC -I$SRC_DIR/Utilities -I$SRC_DIR/Source -I$SRC_DIR/Source/LexerParser -I$SRC_DIR/Source/CTest -I$SRC_DIR/Source/CPack -isystem $SRC_DIR/Utilities/std -isystem $PREFIX/include  -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -mcpu=power8 -mtune=power8 -mpower8-fusion -mpower8-vector -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -pipe  -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/cmake-3.16.0 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -O3 -DNDEBUG   -std=gnu++1z -o CMakeFiles/CTestLib.dir/CTest/cmCTestBuildCommand.cxx.o -c $SRC_DIR/Source/CTest/cmCTestBuildCommand.cxx
[ 87%] Building CXX object Source/CMakeFiles/CTestLib.dir/CTest/cmCTestCoverageCommand.cxx.o
@                                                                                                
**To Reproduce**
./open-ce/open-ce/open-ce build env --docker_build --python_versions 3.8 --build_types cuda open-ce/envs/opence-env.yaml 2>&1 | tee opence-docker-cuda.out

**Expected behavior**
A successful build.

**Execution Logs**
I need to figure out how to attach the console output file opence-docker-cuda.out

**System (please complete the following information):**
 - OS: rhel 7.6alt
 - Architecture: Power  (AC922)

**Additional context**
Add any other context about the problem here.

--- a/envs/lightgbm-env.yaml
+++ b/envs/lightgbm-env.yaml
@@ -5,4 +5,5 @@ packages:
     git_tag : 26e3ecb4156c14d90a66fd1433d52a1d7e27946d
     patches :
       - ../feedstock-patches/cmake/0001-Fix-test.patch
+      - ../feedstock-patches/cmake/0002-set-rhash-version.patch
     runtime_package : False

--- a/feedstock-patches/cmake/0002-set-rhash-version.patch
+++ b/feedstock-patches/cmake/0002-set-rhash-version.patch
@@ -1,0 +1,22 @@
+diff --git a/recipe/meta.yaml b/recipe/meta.yaml
+index dd20d84..d37d3e2 100644
+--- a/recipe/meta.yaml
++++ b/recipe/meta.yaml
+@@ -28,7 +28,7 @@ requirements:
+     - xz              # [unix]
+     - zlib            # [unix]
+     - libuv           # [unix]
+-    - rhash           # [unix]
++    - rhash 1.4.0          # [unix]
+ 
+   run:
+     - bzip2           # [unix]
+@@ -38,7 +38,7 @@ requirements:
+     - xz              # [unix]
+     - zlib            # [unix]
+     - libuv           # [unix]
+-    - rhash           # [unix]
++    - rhash 1.4.0          # [unix]
+     - vs2015_runtime  # [win]
+   
+   run_constrained:

